### ISSUE TITLE
Steam from coal boilers don't fill BC pipe

### ISSUE THREAD

**Issue by KrisBigK** [GregTechCE/GregTech#225]
![2018-07-13_17 07 07](https://user-images.githubusercontent.com/34199560/42683214-469ea98c-86bf-11e8-81ee-36ecf88568ec.png)
Pipe in the background.

Mod list:
jei_1.12.2-4.11.0.202
gregtech-1.12.2-0.4.5.9
CodeChickenLib-1.12.2-3.2.0.345-universal
industrialcraft-2-2.8.91-ex112
buildcraft-7.99.17
forge1.12.2-14.23.4.2705

**Comment by AEnterprise** [comment]
It seems this is not fully a BC issue but rather just how the machine works (and this will be a problem for outputting directly to other modded pipes as well)

it seems the boiler (and maybe other machines) only try to push fluids every 5 ticks, so the combined generation of 5 ticks is more then what a pipe accepts each tick

this burst outputting means for any kind of modded pipes that do not have a transfer rate of 5x the rate it generates at it won't accept it all (unless said pipe extracts from the machine, then it can do it every tick), in this case a buffer is just needed to then extract from that with other pipes

**Comment by EmosewaPixel** [comment]
I've tested the fluid transferring methods of 15 different mods (Mekanism, Thermal Dynamics, GalactiCraft, Extra Utilities, EnderIO, Immersive Engineering, Embers, Translocators, XNet, Actually Additions, FactoryTech, PneumatiCraft, Extra Cells 2, Integrated Dynamics and Tinker's Construct) and all of them worked fine. Perhaps instead of changing the way GTCE outputs, you should change how BC inputs.

**Comment by EmosewaPixel** [comment]
Actually, wait a moment, BC pipes work just fine.
![image](https://user-images.githubusercontent.com/25333164/42878749-8074e41e-8a8e-11e8-9057-390b896f3da3.png)
It's only slow when you use wooden pipes, which you don't need since the boiler auto-outputs.

**Comment by KrisBigK** [comment]
@EmosewaPixel The pipe _will_ output steam, but steam will pile up in the boiler, causing the efficiency to decrease.

**Comment by Archengius** [comment]
tl;dr just use buffer tank
